src/org/rust/lang/core/lexer/RustTokenElementTypes.java

```diff
@@ -148,4 +148,10 @@
         com.intellij.psi.TokenType.WHITE_SPACE
     );
 
+    TokenSet COMMENTS_TOKEN_SET = TokenSet.create(
```

Comment:
Yes, but the parser definition is not useable without an instance. I'm can reuse this one in the parser definition though.